test/edu-benefits/1995/00-required.e2e.spec.js

```diff
@@ -26,12 +26,12 @@ module.exports = E2eHelpers.createE2eTest(
     client.click('.form-progress-buttons .usa-button-primary');
     E2eHelpers.expectNavigateAwayFrom(client, '/veteran/information');
 
-    // Benefits eligibility
+    // Benefits eligibility page.
     client
       .waitForElementVisible('label[for="root_benefit"]', Timeouts.slow);
-    EduHelpers.completeBenefitsSelection(client, testData.benefitSelection.data);
+    EduHelpers.completeBenefitsSelection(client);
     client.click('.form-progress-buttons .usa-button-primary');
-    E2eHelpers.expectNavigateAwayFrom(client, '/benefits/elibility');
+    E2eHelpers.expectNavigateAwayFrom(client, '/benefits/eligibility');
```

Comment:
Ha...well`expectNavigationAwayFrom` just checks that the current URL does not equal the one passed in!